### PR TITLE
Targets in map

### PR DESCRIPTION
**Pull request by RCarmona53**
#### Board:
* [RSLC-3](https://trello.com/c/CWAM22VA/14-rslc-3-as-a-user-i-should-be-able-to-see-all-my-targets-in-the-map-so-i-can-easily-identify-them)
---
#### Description:
As a user I should be able to see all my targets in the map so I can easily identify them

### CODE REVIEW

**Review by enzofab91 — APPROVED** [review]
💯

File: spec/requests/api/v1/targets/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  it 'returns the targets in the correct format' do
+    subject
+    expect(json[:targets].first[:title]).to eq(first_target.title)
+    expect(json[:targets].first[:radius]).to eq(first_target.radius)
+    expect(json[:targets].first[:lat].round(10)).to eq(first_target.lat.round(10))
+    expect(json[:targets].first[:lng].round(10)).to eq(first_target.lng.round(10))
+    expect(json[:targets].first[:topic_id]).to eq(first_target.topic_id)
+  end
+end
```
Comment (diego-suarez) [inline spec/requests/api/v1/targets/index_spec.rb:26]: wdyt about a scenario where there are also targets from other users on the database, and you ensure that only the ones from the user are returned? as a safety check

**Review by diego-suarez — APPROVED** [review]
Great work!

File: spec/requests/api/v1/targets/index_spec.rb (hunk 1 above)
Comment (RCarmona53) [reply spec/requests/api/v1/targets/index_spec.rb:26]: Done!

**Pull request merged**